Service/Microsoft.Health.DeIdentification.Web/Controllers/FhirController.cs

```diff
@@ -107,7 +111,7 @@ public async Task<IActionResult> GetDeIdentificationJobStatus(string operationId
             }
             else
             {
-                string url = GenerateUrl(operationId);
+                string url = $"{Request.HttpContext.Request.Scheme}://{host}/base/operation/{operationId}";
```

Comment:
we can use a GenerateUrl function with request and operation URL passed as parameters.

Reply:
I will restore it